Aplib.Core/Tactics/AnyOfTactic.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Aplib.Core.Tactics
+{
+    /// <summary>
+    /// Represents a tactic that executes any of the provided sub-tactics.
+    /// </summary>
+    public class AnyOfTactic : Tactic
+    {
+        /// <summary>
+        /// Gets or sets the sub-tactics of the tactic.
+        /// </summary>
+        protected LinkedList<Tactic> SubTactics { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AnyOfTactic"/> class with the specified sub-tactics.
+        /// </summary>
+        /// <param name="subTactics">The list of sub-tactics.</param>
+        public AnyOfTactic(List<Tactic> subTactics)
+        {
+            SubTactics = new();
+
+            foreach (Tactic tactic in subTactics)
+            {
+                _ = SubTactics.AddLast(tactic);
+            }
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AnyOfTactic"/> class with the specified sub-tactics and guard condition.
+        /// </summary>
+        /// <param name="subTactics">The list of sub-tactics.</param>
+        /// <param name="guard">The guard condition.</param>
+        public AnyOfTactic(List<Tactic> subTactics, Func<bool> guard) : this(subTactics) => Guard = guard;
+
+        /// <inheritdoc/>
+        public override List<PrimitiveTactic> GetFirstEnabledActions()
```

Comment:
I think this is done weirdly in Java aplib, and we should handle this differently. Semantically the `AnyOfTactic` could be renamed to `RandomTactic`, where a random `Tactic` is picked from a list of `Tactic` instances. However, this is currently not the implementation of this class.

In Java aplib, they retrieve all enabled actions (which are not even actions in reality) and choose an `Action` at random within the `Agent` class. Why should the `Agent` class handle this? Mind you, this is ONLY the case for the `AnyOfTactic`, since the other `Tactic` subclasses only yield a single `PrimitiveTactic` with this method.

Therefore, I propose to...
1. Pick an `Action` at random within this method. This is functionality exclusive to the `AnyOfTactic` class.
2. Immediately invoke the correct `Action`. After addressing point 1, this method would retrieve a single `PrimitiveTactic`. The only place this method is used is the moment that the `Action` corresponding with that `PrimitiveTactic` is invoked. So why not do that here? This is also in line with the semantics of a `Tactic`. It should simply worry about what subtactics it consists of and how to invoke them correctly (which is practically the only thing the three subclasses differ at).